confirm/confirm.go

```diff
@@ -20,17 +20,19 @@ func Ask(s string) bool {
 	reader := bufio.NewReader(os.Stdin)
 
 	for {
-		fmt.Printf("%s [y/n]: ", color.YellowString(s))
+		fmt.Printf("%s [y/N]: ", color.YellowString(s))
 
 		response, err := reader.ReadString('\n')
 		if err != nil {
 			log.Fatal(microerror.Mask(err))
 		}
 
 		switch strings.ToLower(strings.TrimSpace(response)) {
-		case "y", "yes":
+		case "y", "yes", "Y", "YES", "Yes":
 			return true
```

Comment:
As you convert the string to be tested to lower case, there should be node need to test `"Y", "YES", "Yes"` , right?

Reply:
You're right, I didn't catch that. Thank you 👍